f/fcs/fn_fcsFailure.sqf

```diff
@@ -0,0 +1,44 @@
+// F3 - FCS/Commander's Override system
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+// Originally from: https://github.com/NikkoJT/njt_fcs (integrated with permission)
+
+// This function is used in the Hit EH defined in fcs_init. It determines whether a given hit causes an FCS failure.
+
+params ["_target","_projectile","_isDirect"];
+
+	private _projectileClass = "";
+	private _warheadType = "";
+	private _randomNumber = 0;
+
+	// Determine whether the projectile is an anti-tank weapon
+	_projectileClass = typeOf _projectile;
+	if !(_projectileClass isKindOf ["Default",configFile >> "CfgAmmo"]) exitWith {
+		//diag_log "FCS: hit by something other than a weapon" 
+	};
+	_warheadType = (configProperties [configFile >> "CfgAmmo" >> _projectileClass >> "warheadName"]) select 0;
+	if !(_warheadType in ["AP","HEAT"]) exitWith { 
+		// diag_log "FCS: projectile not HEAT or AP" 
+	};
+	if !(_isDirect) exitWith { 
+		// diag_log "FCS: splash damage only" 
+	};
+
+	// 5% of qualifying hits should trigger this
+	_randomNumber = floor (random 20);
+	if !(_randomNumber == 10) exitWith { 
+		// diag_log "FCS: projectile not randomly selected" 
+	};
+	if (_target getVariable ["fcs_failure",false]) exitWith { 
```

Comment:
Might make sense to move this to be the first check

Reply:
Done